Resources/public/js/views/services/ez-subitemboxviewservice.js

```diff
@@ -0,0 +1,114 @@
+/*
+ * Copyright (C) eZ Systems AS. All rights reserved.
+ * For full copyright and license information view LICENSE file distributed with this source code.
+ */
+YUI.add('ez-subitemboxviewservice', function (Y) {
+    "use strict";
+    /**
+     * Provides the view service component for the location view
+     *
+     * @module ez-subitemboxviewservice
+     */
+    Y.namespace('eZ');
+
+    /**
+     * Subitem box view service.
+     *
+     * Loads the models needed by the subitem box view
+     *
+     * @namespace eZ
+     * @class subitemBoxViewService
+     * @constructor
+     * @extends eZ.ViewService
+     */
+    Y.eZ.SubitemBoxViewService = Y.Base.create('subitemBoxViewService', Y.eZ.ViewService, [], {
+        /**
+         * Loads the location, the content and the path for the location id
+         * available in the request and calls the next callback once it's done.
+         *
+         * @method _load
+         * @protected
+         * @param {Function} next
+         */
+        _load: function (next) {
+            var request = this.get('request'),
+                service = this,
+                tasks,
+                endLoad;
+
+            tasks = new Y.Parallel();
+            endLoad = tasks.add();
+            this.search.findContent({
+                viewName: 'search-',
+                loadLocation: true,
+                loadContentType: true,
+                query: {
+                    "LocationIdCriterion": request.params.id,
+                },
+                limit: 1,
+                offset: 0
+            }, Y.bind(function (error, result) {
+                if (!error) {
+                    this.set('content', result[0].content);
+                    this.set('contentType', result[0].contentType);
+                    this.set('location', result[0].location);
+                } else {
+                    this._error('Fail to load subitems');
+                }
+                endLoad();
+
+            }, this));
+
+            tasks.done(function () {
+                service.get('response').view = {
+                    location: service.get('location'),
+                    content: service.get('content'),
+                };
+                next(service);
+            });
+        },
+
+        _getViewParameters: function () {
+            return {
+                content: this.get('content'),
+                contentType: this.get('contentType'),
+                location: this.get('location'),
+            };
+        },
+    }, {
+        ATTRS: {
+            /**
+             * The viewed location
```

Comment:
actually, it is the parent Location of the subitems. This Location is not really *viewed*